packages/side-nav/src/SideNavGroup.tsx

```diff
@@ -5,9 +5,10 @@ import { uiColors } from '@leafygreen-ui/palette';
 import { ulStyleOverrides, SIDE_OFFSET } from './styles';
 
 const sideNavLabelStyle = css`
-  color: ${uiColors.green.base};
+  font-size: 11px;
+  font-weight: bold;
```

Comment:
Thank you for the drive-by on behalf of legacy less code

Reply:
👍